app/components/channel_list/channel_list.js

```diff
@@ -111,39 +115,34 @@ class ChannelList extends React.Component {
         return data.findIndex((obj) => obj.display_name === displayName);
     };
 
-    getAboveAndBelow = (index) => {
-        const channel = this.state.dataSource.getRowData(0, index);
-        const result = buildDisplayNameAndTypeComparable(channel).localeCompare(buildDisplayNameAndTypeComparable(this.props.currentChannel));
-        if (result < 0) {
-            return {above: true, below: false};
-        } else if (result > 0) {
-            return {above: false, below: true};
-        }
-        return {above: false, below: false};
-    };
-
     updateUnreadIndicators = (v) => {
         let showAbove = false;
         let showBelow = false;
 
-        if (this.firstUnreadChannel) {
-            const index = this.getRowIndex(this.firstUnreadChannel);
-            if (index >= 0 && !v.s1[index]) {
-                showAbove = this.getAboveAndBelow(index).above;
+        if (v.s1) {
```

Comment:
What's `v` and `v.s1`? Could they be changed to something more obvious, or are they from RN?